be/src/common/config.h

```diff
@@ -1270,11 +1273,11 @@ CONF_Double(datacache_scheduler_threads_per_cpu, "0.125");
 CONF_Bool(datacache_tiered_cache_enable, "false");
 // Whether to persist cached data
 CONF_Bool(datacache_persistence_enable, "true");
-// DataCache engines, alternatives: starcache.
+// DataCache engines, alternatives: starcache, lrucache, cachelib.
 // `cachelib` is not support now.
-// Set the default value empty to indicate whether it is manully configured by users.
+// Set the default value empty to indicate whether it is manually configured by users.
 // If not, we need to adjust the default engine based on build switches like "WITH_STARCACHE".
-CONF_String(datacache_engine, "");
+CONF_String_enum(datacache_engine, "", ",starcache,lrucache,cachelib");
 // The interval time (millisecond) for agent report datacache metrics to FE.
```

Comment:
Is it necessary to add the `cachelib` as an alternative engine? Because we have deprecated the related library and can not support this engine now.

Reply:
Good suggestion, i will remove it.